sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -11360,4 +11360,61 @@ DROP FUNCTION IF EXISTS create_additional_healthconditions();
 
 INSERT INTO schema_version (version_number, comment) VALUES (458, 'Permanent Deletion | Immunization | healthconditions_id violates not-null constraint error #8983');
 
+-- 2022-05-10 Add reason for deletion to confirmation dialogue - #8162
+ALTER TABLE cases ADD COLUMN deletereason character varying(512);
```

Comment:
Change the column names here as well, change the data type of the `deletionreason` columns to `varchar(255)` and the data type of the `otherdeletionreason` columns to `text`.

Reply:
changed